### PR TITLE
Update german translation

### PR DESCRIPTION
**Pull request by dreua**
I translated all the strings but many strings remain untranslated in the app anyway:

Tooltips

- Save as
- Rotate left/right
- Zoom in/out

Main Menu

- Import
- Save
- Save As
- Export selection
- Zoom in/out
- Edit properties

Edit Submenu and Context Menu

- Duplicate
- Split pages

### Todo

- [x]  Fix untranslated strings
- [x]  Access keys (underscores)

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -11,9 +11,7 @@ msgstr ""
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2020-04-11 10:54+0200\n"
 "PO-Revision-Date: 2019-07-18 20:51+0200\n"
-"Last-Translator: O. Giesmann <nutzer3105@gmail.com>\n"
-"Language-Team: German (http://www.transifex.com/logari81/pdfshuffler/"
```
Comment (dreua) [inline po/de.po:15]: We don't use transifex any more, do we?

File: po/de.po [hunk 2]
```diff
@@ -144,15 +142,15 @@ msgstr "Speichern"
 #: data/pdfarranger.ui.h:3
 #, fuzzy
 msgid "Save As"
-msgstr "Speichern als"
+msgstr "Speichern unter"
```
Comment (dreua) [inline po/de.po:145]: I think that is more common

File: po/de.po [hunk 3]
```diff
@@ -238,27 +236,27 @@ msgstr "Seiten teilen"
...
 
 #: data/menu.ui.h:19
 msgid "Paste Interleave _Odd"
-msgstr ""
+msgstr "Verzahnt einfügen ungerade"
 
 #: data/menu.ui.h:20
 msgid "Paste Interleave _Even"
-msgstr ""
+msgstr "Verzahnt einfügen gerade"
```
Comment (dreua) [inline po/de.po:259]: @timoschwarzer These are the best translations I could come up with, maybe you have got better ideas for them? I am native speaker but I do 99% of computer related stuff in english (not counting [heise.de news](https://heise.de) of course :wink: )

**Comment by dreua** [comment]
Removing the "fuzzy" lines fixed the problem

**Comment by jeromerobert** [comment]
Merged in 2847ba9

**Pull request closed without merging**